tests/acceptance/features/coreApiTrashbin/trashbinSharingToShares.feature

```diff
@@ -1,5 +1,8 @@
 @api
 Feature: using trashbin together with sharing
+  As a user
+  I want to delete resources shared with me
+  So that sharer can restore the deleted resources
```

Comment:
I think we need @phil-davis 's help to suggest good feature stories :smile: . Or anyone?
We are forcing feature story in each feature file, maybe we don't have to :thinking: ?

Reply:
I think this should be from the perspective of Sharer
```
  As a user
  I want the deletion of the resources that I shared to end up in my trashbin
  So that I can restore the resources that were accidentally deleted
```

Reply:
not in this feature only but also in others as well. I feel not so right about the feature description :sweat_smile:

Reply:
@SwikritiT Please, go through other features also maybe you could suggest proper description

Reply:
Swikriti's suggestion sounds good.